docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+.. _conf-web-resources:
+
+.. highlight:: yaml
+
+Conference websites
+===================
+
+Each conference has a section of the website, which is part of the same
+repository as the main conference website. Many parts of the conference
+websites are data-driven, like the generation of the schedule.
+
+.. note::
+    This documentation applies only for conferences from 2020 onwards.
+
+File locations
+--------------
+
+The important locations of files are:
+
+* ``_data/<shortcode>-<year>-config.yaml`` contains the general config file
+  for rendering a specific conference. This includes settings like the
+  conference dates, sponsors and ticket prices.
+* ``_data/<shortcode>-<year>-sessions.yaml`` contains the conference sessions,
+  the main talks on the agenda. For each session, it includes things like the
+  title and abstract, and name and other info for the speaker(s).
+* ``_data/<shortcode>-<year>-schedule.yaml`` contains the conference schedule.
+  This includes the talks, but also items like "Lunch".
+* ``conf/<shortcode>/<year>/`` contains the RST files used for rendering the
+  conference website. Copy these over from the previous conference
+  (chronologically, not the previous conference in the same location).
+  You may need to update some reference to specific locations/years,
+  and keep an eye on any FIXME's or TODO's.
+* ``include/conf/`` contains text snippets which are generally the same
+  between conferences, or at least between conferences in the same location.
+  Examples include the description of lightning talks, or the Portland hike.
+* ``_templates/<year>/`` and ``_templates/include/`` contain HTML or RST
+  templates used in rendering the conference pages. This includes the
+  website design, navigation and snippets used in schedule rendering.
+
+The YAML files are validated using the schemas in ``_data/schema-*``.
```

Comment:
Passive voice alert! Who/what is doing the validation? Is it something a human needs to do or is it automated?